ingestion/src/metadata/ingestion/ometa/mixins/glossary_mixin.py

```diff
@@ -0,0 +1,44 @@
+"""
+Mixin class containing Glossaries specific methods
+
+To be used be OpenMetadata
+"""
+import logging
+from typing import List, Optional, Type, TypeVar
+
+from pydantic import BaseModel
+
+from metadata.generated.schema.entity.data.glossary import Glossary
+from metadata.ingestion.ometa.client import APIError
+
+T = TypeVar("T", bound=BaseModel)  # pylint: disable=invalid-name
+logger = logging.getLogger(__name__)
+
+
+class GlossaryMixin:
+    def list_glossaries(
```

Comment:
We already have a `list_entities` function. Doesn't it work for `Glossary`?

Reply:
I have removed the method as discussed and can be done using `list_entities`